ui/lib/config-ui/addon/components/messages/message-expiration-date-form.hbs

```diff
@@ -0,0 +1,54 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<div class="has-top-padding-xs has-bottom-padding-s is-narrow is-flex-align-baseline">
+  <RadioButton
+    class="radio"
+    name={{@attr.name}}
+    id="never"
+    value="never"
+    @value="never"
+    @onChange={{fn (mut @message.endTime) ""}}
```

Comment:
Should choosing "never" also clear the endTime? Or maybe we put that logic in the serializer so that switching between the two options keeps the previous date selection.

Reply:
Good point, I think it should keep the previous date selection. Instead of doing this in the serializer I have a tracked property in the component and have an action that sets it in the `create-and-edit-form`.